### ISSUE TITLE
ngx-mqtt throws warning when installing with angular 10

### ISSUE THREAD

**Issue by guuurris** [sclausen/ngx-mqtt#166]
- [x] **I have searched for similar issues in this repository, but couldn't find one.**
- [x] **I have read the README and have a basic understanding how angular works.**
## I'm submitting a...

- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report  
- [ ] Feature request

## Current behavior

I can not use this library with any version of Angular 10
```
npm WARN ngx-mqtt@7.0.14 requires a peer of @angular/common@^9.1.12 but none is installed. You must install peer dependencies yourself.
```
## Expected behavior

To change peerDependencies to allow any version above Angular 9
## Minimal working example

Example has a package.json configuration but as the terminal can not be used inside of the codesandbox it is a difficult to see the actual error from there

 [![Edit serverless-water-zti7o](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/serverless-water-zti7o?fontsize=14&hidenavigation=1&module=%2Fpackage.json&theme=dark)

## What is the motivation / use case for changing the behavior?
To make it work for further development

## Environment

<pre><code>
ngx-mqtt version: 7.0.14

</code></pre>

**Comment by sclausen** [comment]
Would you mind opening a PR for this `npm WARN`? I'm a bit busy right now. And after all, it's just a warn, not an error that prevents using it.

**Comment by guuurris** [comment]
PR created (https://github.com/sclausen/ngx-mqtt/pull/167) not able to link to this issue

**Comment by sclausen** [comment]
should be fixed by #167
